### ISSUE TITLE
Alephium >ALPH< Missing

### ISSUE THREAD

**Issue by ezio2code** [todxx/teamredminer#631]
I tired to dualmine ETH+Alph but noticed that it's now missing from the dual coin dropdown menu.
Is it coming back to your miner ?

**Comment by happyandygod** [comment]
RIP TON
we need ALPH !!!

**Comment by ishiro13** [comment]
Make pls ALPH we need him =)

**Comment by UselessGuru** [comment]
> missing from the dual coin dropdown menu

What menu?
Dual mining ETH+Alph is still supported in TRM 0.10.1

**Comment by UselessGuru** [comment]
> it's now missing from the dual coin dropdown menu

What dropdown menu?

**Comment by ishiro13** [comment]
> > it's now missing from the dual coin dropdown menu
> 
> What dropdown menu?

i think this
![opera_tDV3w1FwdF](https://user-images.githubusercontent.com/14308554/174408663-5b2ff5ce-38dc-4abe-872b-9493cef95212.png)

**Comment by ezio2code** [comment]
When selecting a second coin to mine Alph is no longer there.

![09D31AEB-F642-419D-AEB4-EAC25A75659E](https://user-images.githubusercontent.com/13763973/174423165-61907c1c-ce0f-4376-8d99-37a4ea67a7ef.jpeg)


Guys come on we are paying customers, theres a day left of ton mining and we need to be prepared to make the switch to alph.
I have over 5GH and I am starting to get panic attacks thinking that I will have to use lolminer.

Please help us out.

**Comment by CamJava007** [comment]
I must agree with all of this.  ALPH is no longer even in the USAGE.txt under algorithms.  It would be great to have it back.  I'd really rather not have to move to another program.  I've been Team Red for years.

**Comment by ishiro13** [comment]
> Я должен согласиться со всем этим. ALPH больше нет даже в USAGE.txt под алгоритмами. Было бы здорово вернуть его. Я действительно предпочел бы не переходить на другую программу. Я много лет был в Красной Команде.

Yep TRM its TOP for AMD

**Comment by standula** [comment]
any news ? about ALPH

**Comment by Karaguay** [comment]
we need eth + alph

**Comment by grenelt** [comment]
... and ETC + ALPH! ;)

**Comment by ezio2code** [comment]
Any updates if the miner will support alph dual mining again ?

**Comment by Karaguay** [comment]
you can use lolminer

**Comment by estebanflore** [comment]
Buen dia, desarrollar el dual mining con alph o kaspa, no quiero usar otros mineros, teamredminer el mejor para amd

**Comment by InTeamLife** [comment]
hello team, you need to develop dual mining!!!

**Comment by estebanflore** [comment]
Si por su puesto, necesitamos más opciones para sobrevivir en este mercado.
